README.md

```diff
@@ -62,23 +55,14 @@ In order to use this tool, your system needs to fulfill the following prerequisi
 * Software
     * Linux
       * [bash](https://www.gnu.org/software/bash/) >= 4.2
```

Comment:
Do we really need bash? If we are honest it should be only tar or do we shell calls from Python